store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9262,14 +9262,20 @@ TODO: delete them permanently from here
 <attr id="2096" name="zimbraSieveFeatureVariablesEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.7.6" deprecatedSince="8.7.8">
   <globalConfigValue>FALSE</globalConfigValue>
   <desc>Whether to enable the Sieve "Variables" extension defined in RFC 5229 in the user-defined sieve rule.</desc>
-  <deprecateDesc>deprecated in favor of zimbraSieveRejectMailEnabled, which can be used at account level</deprecateDesc>
+  <deprecateDesc>deprecated as this attribute is not used anymore and variable feature is availabe to use for all.</deprecateDesc>
```

Comment:
I think only the following description will be sufficient - 
"Deprecated as this attribute is not used anymore"
Whether the variable feature is available for all or not will be part of documentation right?
OR may be you can rephrase as below  - 
"Variable feature is always enabled, hence this attribute has been deprecated"